my-app/src/App.js

```diff
@@ -69,6 +69,8 @@ const Main = () => (
       <Route exact path='/' component={Landing} />
       <Route exact path='/dashboard' component={Dashboard} />
       <Route exact path='/board' component={Board} />
+      <Route path='/board/:name' component={Board} />
```

Comment:
We should go over as a team what the /: does

Reply:
I can explain during our meeting, but here's an explanation for the record:

With all Routes, it basically checks the URL the user went to and see's which Route it matches with. The Route '/board/:name' will match any url that's structured like this:

> teamreflex.com/board/myFirstBoard 

The thing is, it essentially stores that string "myFirstBoard" as a variable called "name". More specifically, you can access it in the component being rendered by the Route, which is Board. Here's an example of me using it in Board.js, line 16-18 currently:

```javascript
if (this.props.match.params.name !== undefined) {
    this.setState({boardName: this.props.match.params.name});
}
```

'this.props.match.params.name' is where I'm accessing that variable. This way we can generate dynamic content based on the URL the user went to.